frontend/src/components/widgets/FileUploader/FileDropzone.test.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import { shallow } from "enzyme"
+import Dropzone from "react-dropzone"
+import FileDropzone, { Props } from "./FileDropzone"
+
+const getProps = (props: Partial<Props> = {}): Props => ({
+  disabled: false,
+  onDrop: jest.fn(),
+  multiple: true,
+  acceptedExtensions: [],
+  maxSizeBytes: 2000000,
```

Comment:
This isn't suppose to be `200MB`, is it? Can we change it to something else? I forgot that I'm looking at a test file, and started counting 0s 😂.